### DISCUSSION TITLE
Please Explain Why  the confusion matrix have background

### DISCUSSION THREAD

**Discussion by suhailshaji** [ultralytics/ultralytics#1666] [Q&A] [answered]
I want to understand I didn't trained any class called background but as I am plotting the confusion matrix it give a class called background why? did it train a separate class called background

**Comment by glenn-jocher** [comment, marked as answer]
@suhailshaji detection models always have the possibility of an FP. This detection truth may not belong to any class, in which case it is assigned to background.
